opentreemap/importer/tasks.py

```diff
@@ -144,7 +144,10 @@ def commit_import_event(import_type, import_event_id):
 
     finalize_task = _finalize_commit.si(import_type, import_event_id)
 
-    chord(commit_tasks, finalize_task).delay()
+    async_result = chord(commit_tasks, finalize_task).delay()
```

Comment:
This worked in testing without saving. I will do some tests that include a celery outage/restart and see how saving affects the situation. If needed, will add.